src/parsers/index.js

```diff
@@ -8,8 +8,7 @@
  */
 
 import {ParseResult} from './parser_nodes';
-import {parse as parseJs} from './babylon_parser';
-import {parse as parseTs} from './typescript_parser';
+import {parseJs, parseTs} from './babel_parser';
```

Comment:
this is not in your change, but while you are right here, can you add support for `mjs` file extension as seen  in jest-community/vscode-jest#551, particularly [here](https://github.com/jest-community/vscode-jest/issues/551#issuecomment-579529020). Hopefully, the optional-chaining issue will be automatically resolved once this PR is merged.

Reply:
i look into it

Reply:
implemented